### ISSUE TITLE
parameter calculation

### ISSUE THREAD

**Issue by padmaksha18** [metalcorebear/COVID-Agent-Based-Model#5]
hi Team, thank you for sharing this code. I had a question on how the parameters are estimated. In the parameters file, all the values are hardcoded. So, is this data obtained from social media somehow or some real world data? Please elaborate.

Thanks,
Padmaksha

**Comment by DemetersSon83** [comment]
Hey. The parameters are estimated based on real world data, or they are simply a best guess. The purpose of the parameters file is to experiment and adjust them to see what works or makes the most sense for the scenario you are trying to model.

**Comment by padmaksha18** [comment]
Hi @metalcorebear , So you mean we change the parameters and compare the model outputs with the real-time data and thus make our model a simulation of the real-world scenario. We fix those parameters of the model which can best interpret the real-world scenario. Is my understanding correct? Also, I had one more question. I wanted to improve these parameters using insights from social media. But in case of covid disease spread, the usual graph network measures like centrality, transitivity ,clustering etc which gives useful intuition for information spread does not provide any hint about disease spread. Can you provide some insights about how can social media be used to improve the simulation model parameters. Thant would be really helpful to me! Thanks.

**Comment by DemetersSon83** [comment]
Yes, that is correct.  Regarding social media, I think it would be interesting to use social media posts to identify potential geographic hot spots.  I think this could be done using NLP to identify posts that convey symptoms or explicitly stating that the individual is COVID-positive.  You could then look at their social media connections that are geographically colocated and see if the same patterns emerge in their posts, and use that as a proxy for disease spread.  It's just an idea that I haven't really explored in any great detail.  Is that aligned with what you were thinking?

**Comment by padmaksha18** [comment]
hi @metalcorebear , thank you for the answer. I am trying to understand the spread of awareness regarding covid testing
from social media and not exactly the spread of the disease. It is very difficult to analyze the spread of the disease from social media data. My purpose is to enhance the performance of the simulation system from social media data. Therefore , i wanted some more details from you about these initial model parameters like Transmission probability,progression_period = Average number of days until a patient seeks treatment, interactions,
reinfection_rate, and all other parameters. Can I get any intuition of these params from social media data? Thanks!

**Comment by DemetersSon83** [comment]
Hi @padmaksha18.  I'm interested in information propagation through social media as well. I've built a meme propagation ABM that might also provide some insight for you.  It can be found here: https://github.com/metalcorebear/Meme-Propagation-Agent-Based-Model

As far as linking meme propagation parameters to infection models, I'm not sure how to approach that.  It is an interesting problem, though, that I will ponder.

**Comment by padmaksha18** [comment]
hi, Thank you for the information. You are right, it is difficult to
relate information spread with real time disease spread. So, I am just
trying to find if any social media features like volume/location of tweets
etc can boost my ABM . Thanks!
